daemon/inspect.go

```diff
@@ -86,6 +89,14 @@ func (daemon *Daemon) getInspectData(container *Container) (*types.ContainerJSON
 	return contJSONBase, nil
 }
 
+// ContainerExecInspect returns low-level information about the exec
+// command. An error is returned if the exec cannot be found.
+//
+// golnit complains exported method ContainerExecInspect returns
+// unexported type *daemon.execConfig, which can be annoying to use
+//
+// not sure what to do about the return value, perhaps we should
+// export execConfig
```

Comment:
Pleaes export execConfig. Since it's returned by an exported function, the type should also be exported.